install.sh

```diff
@@ -262,7 +262,9 @@ nvm_detect_profile() {
 # Node, and warn them if so.
 #
 nvm_check_global_modules() {
-  command -v npm >/dev/null 2>&1 || return 0
+  local NPM_COMMAND
+  NPM_COMMAND="$(command -v npm 2>/dev/null)" || return 0
+  [[ -n "${NVM_DIR}" ]] && [[ "${NPM_COMMAND}" == "${NVM_DIR}"* ]] && return 0
```

Comment:
unfortunately, this is bash-specific syntax, and nvm has to work on all posix shells.

Reply:
Whoops, sorry. I'll see if I can figure out how to do it with parameter expansion.